rosbag2_transport/test/rosbag2_transport/test_play.cpp

```diff
@@ -109,6 +109,10 @@ void spin_thread_and_wait_for_sent_service_requests_to_finish(
 }
 }  // namespace
 
+class RosBag2PlayTestFixtureMessageOrder
+  : public RosBag2PlayTestFixture, public WithParamInterface<rosbag2_transport::MessageOrder>
+{};
```

Comment:
Nitpick: It could be in two lines
```suggestion
class RosBag2PlayTestFixtureMessageOrder
  : public RosBag2PlayTestFixture, public WithParamInterface<rosbag2_transport::MessageOrder> {};
```

Reply:
changed in 80a03242c55ed4a628aef16032331bdab1267b87